src/common/exif.cc

```diff
@@ -1172,7 +1199,7 @@ static bool _exif_decode_exif_data(dt_image_t *img, Exiv2::ExifData &exifData)
     }
 
     // FIXME: Should the UserComment go into the description? Or do we need an extra field for this?
-    if(FIND_EXIF_TAG("Exif.Photo.UserComment"))
+    if(FIND_EXIF_TAG("Exif.Photo.UserComment") || FIND_EXIF_TAG("Exif.Image.ImageDescription"))
```

Comment:
Not sure these are the same thing: https://exiftool.org/forum/index.php?topic=10029.msg52187#msg52187

Also, did you test this? The `Exiv2::CommentValue` is made to specifically handle the 8 bytes of mandatory comment coding prefix and to decode encoded payload of `UserComment`, haven't checked what happens if you feed an ordinary ASCII string of `ImageDescription` to it...

Might be an idea to handle these separately?

Reply:
I didn't find any problems with the current implementation but I agree that it would be better to handle them separately. I'll change that.

I added the _ImageDescription_ because it is similar to _dc:description_ [(IPTC Photo Metadata Mapping Guidelines)](https://iptc.org/std/photometadata/documentation/mappingguidelines/#description). I have no opinion if _UserComment_ should also go into description.

Reply:
Thanks for reviewing.